com.unity.shadergraph/Editor/Drawing/Blackboard/BlackboardProvider.cs

```diff
@@ -247,17 +248,39 @@ void AddBuiltinKeyword(GenericMenu gm, ShaderKeyword keyword)
         void EditTextRequested(Blackboard blackboard, VisualElement visualElement, string newText)
         {
             var field = (BlackboardField)visualElement;
```

Comment:
Just cast field as a `BlackboardFieldView`, you don't need a separate variable since BFV inherits from BF.